README.md

```diff
@@ -15,4 +18,17 @@ results you would get.
 Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
-Add your answers to this markdown file.
+To verify this crazy claim, I would make my own random arbitrary set of arrays of varying
+sizes. For example, a list of size 1 and another of size 10 and another of size 100 up to
+1,000,000. This would provide me with a relatively good sample space to check whether this
+fantastic new sorting algorithm is correct. I could then - ceteris paribus - run the algorithm
+over all of the arrays and then check by graphin the runtimes to see if it fits the $O(n)$ evaluation and has linear growth.
```

Comment:
How would you be able to tell if it "fits the $O(n)$ evaluation"? What would you expect to see?